### PR TITLE
validators directory: Add Citizen Web3

### PR DESCRIPTION
**Pull request by alibabaedge**

### CODE REVIEW

**Comment by VT-Cosmos** [comment]
Thank you for submitting @alibabaedge - The community will get back to you shortly.

**Comment by VT-Cosmos** [comment]
@alibabaedge 
Thank you for your submission.
There were some updated point that are missing from the application.

Unique validator key to submit (from Cosmos Hub)
Region you and/or your validator is based (country)
I suggest that you check the updated requirements in the README to ensure all is covered in the submission, before the deadline.

Also please note that the GenTX submission deadline is Feb 23, 5pm PST / Feb 24, 1am UTC

File: Citizen Web3.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Website: https://www.citizenweb3.com/
+- Email: citizenweb3@protonmail.com
+
+5) Team structure, roles, and Discord handles
+
+- Citizen web3 is made of several parts. The infrastructure part of the project consists of 2 people. Myself, the founder and a devops. The podcast part has 2 more people. And the validator info project has 4 more people working on it. 3 developers and 1 designer. 
+Discord handles: serejandmyself_citizenweb3 | alibabaedge
+
+6) Link to your main social accounts
+
```
Comment (alibabaedge) [inline Citizen Web3.md:26]: 7) Unique validator key to submit (from Cosmos Hub)

- cosmosvaloper1e859xaue4k2jzqw20cv6l7p3tmc378pc3k8g2u (from gaia)

(using different keys)

8) Region you and/or your validator is based (country)

- We are not bound to a single region. Mostly our operations are bound to Western Europe (Portugal)

File: govgen-1/gentx/Citizen Web3.json [hunk 2]
```diff
@@ -0,0 +1 @@
...80597A20ACFE","website":"https://www.citizenweb3.com/staking","security_contact":"","details":"Citizen Web3: Embracing Decentralization, Empowering Communities. The Voice of Web3 \u0026 Non-custodial staking service"},"commission":{"rate":"0.100000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"govgen1m77eksxfz9q50qejnqf720sns7q0xtx8swn0jr","validator_address":"govgenvaloper1m77eksxfz9q50qejnqf720sns7q0xtx8pv6wce","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"l7oO35UQhzr40OCKrK2VZMkw1AjJLNyKZx9wA+rFaN8="},"value":{"denom":"ugovgen","amount":"1000000"}}],"memo":"12bdc497920301a6c8071341e0345213807ea24c@192.168.1.8:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A7DaVJ9ZB3OGiinCrcUTYN+PvVlavLFfau5I2nMi44Sk"},"mode_info":{"single":{"mode":"SIGN_MODE_LEGACY_AMINO_JSON"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["xsk1KCMJdYP7JClzY8TOsSH/KWp7niG2XoSojcurZI19NfF7pkYB6Obo3rUVk+HEHsVWVc4ZmKvOdYxNacxN9A=="]}
```
Comment (albttx) [inline govgen-1/gentx/Citizen Web3.json:1]: Rate and max_rate should be 0

Comment (alibabaedge) [reply govgen-1/gentx/Citizen Web3.json]: Changed

**Pull request merged**